packages/ndla-code/src/Codeblock/Codeblock.tsx

```diff
@@ -84,23 +96,20 @@ export const Codeblock = ({ title, code, format = 'markup' }: Props) => {
         showLineNumbers>
         {code}
       </SyntaxHighlighter>
+
       <Button
-        title="Kopier kode"
+        title={t('codeBlock.copyButton')}
         disabled={isCopied}
         data-copy-string={code}
         onClick={() => {
           copyTextToClipboard(code);
           setIsCopied(true);
         }}>
-        <>
-          {isCopied ? <Done /> : <Copy />}{' '}
-          {isCopied
-            ? 'Kode kopiert til utklippstavle'
-            : 'Kopier kode til utklippstavle'}
-        </>
+        {isCopied ? <Done /> : <Copy />}{' '}
```

Comment:
Tror du kan bare fjerne `{' '}` denne

Reply:
Den lager et lite mellomrom mellom ikonet og teksen sånn at det ser litt finere ut :-) Om det derimot er en fin løsning for å få slått sammen denne linjen og neste si gjerne i fra da kan mellomrommet legges til der, fant ikke noe rask løsning.

Reply:
Å ja, trodde den ikke var i bruk, da er det greit :D